modules/openy_gc_auth/modules/openy_gc_auth_avocado_sso/openy_gc_auth_avocado_sso.info.yml

```
@@ -3,6 +3,6 @@ type: module
 description: 'Avocado integration for Virtual YMCA'
 package: Open Y Virtual YMCA
 version: 1.7.1
-core_version_requirement: ^8 || ^9 || ^10
+core_version_requirement: ^8 || ^9 || ^10 || ^11
```

For all info.yml files in current PR, please

```suggestion
core_version_requirement: ^10 || ^11
```